api/handler.go

```diff
@@ -142,6 +152,57 @@ func getContainerInfoRequest(body io.ReadCloser) (*info.ContainerInfoRequest, er
 	return &query, nil
 }
 
+func getEventRequest(request []string) (*events.Request, bool, error) {
+	if len(request) <= 0 || len(request[0]) == 0 {
+		return events.NewAllEventsRequest(), true, nil
+	} else if len(request) > 1 {
+		return nil, false, errors.New("Should have received one url rawQuery but got multiple")
+	}
+	rawQuery := strings.Trim(request[0], "/ ")
+
+	query := events.NewRequest()
+	getEventsFromAllTime := false
+
+	urlMap, err := url.ParseQuery(rawQuery)
+	if err != nil {
+		glog.Errorf("url parse error of %v is %v", rawQuery, err)
+		return nil, false, err
+	}
+
+	valueMap := make(map[string]urlData)
+	for key, val := range urlMap {
+		// if a variable is assigned twice, ex. &TypeOom=true&TypeOom=false
+		// then only the first will be taken
+		newInt, err := strconv.ParseInt(val[0], 0, 0)
+		if err == nil {
+			valueMap[key] = urlData{intVal: int(newInt)}
+			continue
+		}
+		newBool, err := strconv.ParseBool(val[0])
+		if err == nil {
+			valueMap[key] = urlData{boolVal: newBool}
+		}
+	}
+
+	getEventsFromAllTime = valueMap["allTime"].boolVal
+	query.IncludeSubcontainers = valueMap["IncludeSubcontainers"].boolVal
```

Comment:
These are usually in lowercase with `_` so:
- `subcontainers=true`
- `oom_events=true`
- `creation_events=true`
- `deletion_events=true`
- `start_time=TIME`
- `end_time=TIME`